opentracing-api-extensions-tracer/src/test/java/io/opentracing/contrib/api/tracer/APIExtensionsSpanTest.java

```diff
@@ -207,6 +209,22 @@ public void testOnSetTagBoolean() throws InterruptedException {
         }
     }
 
+    @Test
+    public void testOnSetTagNull() throws InterruptedException {
```

Comment:
I'd split this in `testOnSetTagKeyNull` and `testOnSetTagValueNull` and verify that a `NullPointerException` must be fixed for both with the old code.

Reply:
Agree, once we have settled on the approach for handling null values.